packages/plugins/rubygem-foreman_fog_proxmox/rubygem-foreman_fog_proxmox.spec

```
@@ -40,7 +40,7 @@ Provides: foreman-%{plugin_name} = %{version}
 
 %description
 Foreman plugin adds Proxmox VE compute resource using fog-proxmox. It is
-compatible with Foreman 1.17+.
+compatible with Foreman 1.21+.
```

You can reuse the macro here:
```suggestion
compatible with Foreman %{foreman_min_version}+.
```